internal/slicer/slicer_test.go

```diff
@@ -224,16 +236,198 @@ var slicerTests = []slicerTest{{
 		"/dir/nested/copy-3": "file 0644 84237a05",
 	},
 	report: map[string]string{
-		"/dir/":              "dir 0755 {test-package_myslice}",
-		"/dir/nested/":       "dir 0755 {test-package_myslice}",
 		"/dir/nested/copy-1": "file 0644 84237a05 {test-package_myslice}",
 		"/dir/nested/copy-3": "file 0644 84237a05 {test-package_myslice}",
 		"/dir/text-file-1":   "file 0644 5b41362b {test-package_myslice}",
 		"/dir/text-file-3":   "file 0644 5b41362b {test-package_myslice}",
 	},
+}, {
+	summary: "Copyright is installed",
```

Comment:
I had to remove the copyright and move it to a separate test because testing several packages was getting too complex and magical.

Reply:
Sounds good, thanks for doing that.